### DISCUSSION TITLE
Privacy mode display issue

### DISCUSSION THREAD

**Discussion by yanxin152133** [rustdesk/rustdesk#8996] [General]
### Bug Description

Software version：1.2.7、1.3.0
In version 1.2.7, when privacy mode 1 is checked and connected to remote Windows, the page for entering the lock screen password cannot be displayed normally.
In version 1.3.0, if you check privacy mode 1, the screen will be black.

### How to Reproduce

Turn on privacy mode

### Expected Behavior

The unlock page is displayed normally, and the lock screen password input interface is displayed normally.

### Operating system(s) on local side and remote side

win10 ltsc 1809

### RustDesk Version(s) on local side and remote side

1.2.7->1.2.7    1.3.0->1.3.0

### Screenshots

1.3.0
![image](https://github.com/user-attachments/assets/a7946a20-76a7-4d41-aa88-8cd2366af19b)
![image](https://github.com/user-attachments/assets/98dbc381-221d-4955-b28d-e6ef7119a040)

**Comment by rustdesk** [comment]
You can try out privacy mode 2, though we just fix a bug of mode 2

**Reply by yanxin152133** [reply]
> You can try out privacy mode 2, though we just fix a bug of mode 2

ok,Thanks
